components/centos/centos-mesos-marathon-singlenode-setup/Vagrantfile

```diff
@@ -19,7 +22,10 @@ Vagrant.configure("2") do |config|
         sudo yum -y install epel-release
         sudo yum -y install ansible
         sudo ansible-playbook /vagrant/provisioning/playbook.yml
+        sudo mkdir -p /etc/marathon/conf
+        sudo sh -c "echo \"#{IP_ADDRESS}\" > /etc/marathon/conf/hostname"
+        sudo sh -c "echo \"#{IP_ADDRESS}\" > /etc/mesos-slave/ip"
```

Comment:
Is `/etc/mesos-slave` directory already present to box or you missed to create it like you did for `marathon/conf` ?